spec/factories/bus_stops.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :bus_stop do
+    name 'stop'
```

Comment:
Stops can have the same name?

Reply:
[Yeah, at least that's what's written](https://github.com/umts/stop-project/blob/master/app/models/bus_stop.rb#L7)

Reply:
I don't see how that would be useful though

Reply:
@sherson, any thoughts about whether we can expect stops to have unique names coming from Hastus?

Reply:
Okay, so sounds like we should allow for now.